README.md

```diff
@@ -51,6 +53,12 @@ There are `make` commands that alias some of this functionality:
  * `publish` -- Outputs the specification as a **single file** into the `build/` directory
  * `serve` -- Serves a preview of the specification in human-readable format
 
+### Running locally
+To run the swagger api into http://0.0.0.0:5000, the you need to execute:
+ 1. `make install`
+ 2. `make build` created the build into `/build` directory
```

Comment:
Running this gives me 

```
make: Nothing to be done for `build'.
```

Did you miss something from the makefile, or is a different command?